internal/pkg/exec/docker.go

```diff
@@ -39,6 +43,15 @@ type BuildArguments struct {
 	Args       map[string]string // Optional. Build args to pass via `--build-arg` flags. Equivalent to ARG directives in dockerfile.
 }
 
+type dockerConfig struct {
+	CredsStore  string            `json:"credsStore,omitempty"`
+	CredHelpers map[string]string `json:"credHelpers,omitempty"`
+}
+
+const(
+	CredStoreECRLogin = "ecr-login" // set on `credStore` attribute in docker configuration file
```

Comment:
```suggestion
	credStoreECRLogin = "ecr-login" // set on `credStore` attribute in docker configuration file
```
Maybe we don't need to make it public